src/lib/ARBodyTrackingBool.h

```diff
@@ -0,0 +1,11 @@
+//
+//  ARBodyTrackingBool.h
+//  example-Matte
+//
+//  Created by Pierre Tardif on 09/04/2020.
+//
+
+// Comment those to add/remove depth Body Tracking
+#ifndef ARBodyTrackingBool_h
```

Comment:
I don't know how necessary this actually is. I think it would be better to take this out and utilize the `ARBodyTrackingConfiguration.isSupported` call instead and rely on people setting up the appropriate session type. 

(I know Face tracking is doing something similar; that should be taken out as well at some point as well if a similar call exists; but that's not something you need to worry about)

Reply:
yes of course, just tried to follow the face one.

I have pushed on my branch of develop, shall i ask for pull again?

++